sympy/core/tests/test_relational.py

```diff
@@ -1187,3 +1188,94 @@ def _eval_is_ge(lhs, rhs):
     assert is_gt(PowTest(3, 9), PowTest(3,2))
     assert is_le(PowTest(3, 2), PowTest(3,9))
     assert is_lt(PowTest(3, 2), PowTest(3,9))
+
+def test_issue_20584():
+    # TODO: cases with multiple valid results should be removed once the
+    #  relations evaluate to the most simplified values (first in valid,
+    #  if present)
```

Comment:
I think it's better just to assert the current behaviour and use a comment to indicate what the correct result should be e.g.:
```python
xf = Symbol('xf', finite=True)
assert (abs(xf) - oo).is_extended_negative is None  # Ideally this would be True
```

Reply:
Then if it gets fixed in future the test will fail but it will be clear that it can be changed from `None` to `True`.